### ISSUE TITLE
Correct mistaken district-level indicator numbers

### ISSUE THREAD

**Issue by nfra** [jvargh7/nfhs5_factsheets#4]
The districts **Mahisagar, Gujarat (GJ)** and **Warda, Maharashtra (MH)** have the wrong indicator numbers. They are missing an indicator numbered as 10. Instead, they skip from 9 to 11.

The misnumbering ends at the end of the first page's table, at 32. The next page starts at a different indicator labeled 32 as well, which aligns with the other district factsheets.

**Comment by jvargh7** [comment]
Good catch Nathan. I extracted as-is from the PDFs. Will work on this later today.
